lms/urls.py

```diff
@@ -609,9 +609,13 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-    urlpatterns += static(
-        settings.PROFILE_IMAGE_DOMAIN + settings.PROFILE_IMAGE_URL_PATH, document_root=settings.MEDIA_ROOT
-    )
+    # DO NOT MERGE this configuration should not be conditional, even in devstack/DEBUG,
+    # but it's here for the moment as it permits validating using the s3 backend in devstack.
+    if 'location' in settings.PROFILE_IMAGE_BACKEND_CONFIG['storage_kwargs']:
```

Comment:
my thinking was to get rid of this condition entirely since it's all subject to `DEBUG` which indicates development mode.  I just needed to work around it for the purposes of exercising the aws code in this PR.  No sane production configuration would serve files using `static`.  That said - I'm all in favor of improving the naming.